privaterelay/settings.py

```diff
@@ -22,10 +22,25 @@
 from hashlib import sha512
 import base64
 
+
+# This needs to be before markus, which imports pytest
+IN_PYTEST = "pytest" in sys.modules
+if IN_PYTEST:
+    assert 1==2
```

Comment:
question: I don't understand why we're doing this? This looks like it's just going to fail everything when we're "in pytest"?

Reply:
That is because I didn't remove my test code from the PR 😅 
Also, I should move this branch to the main repo so CI will run the tests.